cmd/minikube/cmd/start_flags.go

```diff
@@ -91,6 +91,7 @@ const (
 	dryRun                  = "dry-run"
 	interactive             = "interactive"
 	waitTimeout             = "wait-timeout"
+	startHostTimeout        = "starthost-timeout"
```

Comment:
we already have a waitTimeout flag "wait-timeout"
it might have been unued. make sure it is used 

and do you mind sharing the details of a case that you needed this flag ? I am curious, if this would make sense to increase it for everyone by default?

Reply:
The use case is described in the issue #8897. Quoted (not mine):
```
My actual use case is on Hyper-V, when creating a disk image for 128 GBs takes more than 240 seconds (the hardcoded timeout), and thus the whole minikube start fails.
```

(Basically what you said in your last comment.)